common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/JSUtil.java

```diff
@@ -171,11 +179,12 @@ public static String getTime(long time) {
             ret.append(min).append("mn ");
         } else if (min > 0) {
             ret.append(min).append("mn ");
-        } else {
+        } else if (sec > 0) {
             ret.append(sec).append("s ");
         }
 
-        ret.append("ago");
+        if (seconds > 0)
```

Comment:
Please use brackets with _if_ for maintainability.

If we go back and add a second statement to the _if_ or _else_ clause without remembering to add the curly braces, the code will break in unexpected and amusing ways.